devices.js

```diff
@@ -1441,6 +1459,31 @@ const devices = [
             tz.tuya_thermostat_force, tz.tuya_thermostat_preset,
         ],
     },
+    {
+        fingerprint: [{modelID: 'v90ladg\u0000', manufacturerName: '_TYST11_wv90ladg'}],
+        model: 'HT-08',
+        vendor: 'ETOP',
+        description: 'Wall-mount thermostat',
+        supports: 'thermostat, temperature',
+        fromZigbee: [
+            fz.tuya_thermostat_weekly_schedule,
+            fz.etop_thermostat,
+            fz.ignore_basic_report,
+        ],
+        toZigbee: [
+            tz.etop_thermostat_system_mode,
+            tz.etop_thermostat_away,
+            tz.tuya_thermostat_child_lock,
+            tz.tuya_thermostat_current_heating_setpoint,
+            tz.tuya_thermostat_weekly_schedule,
+        ],
+        onEvent: tuya.setTime,
+        meta: {
+            weeklyScheduleMaxTransitions: 4,
```

Comment:
Can you wrap this in `thermostat`?

`meta: {thermostat: {weeklyScheduleSupportedModes, ....}}`